article_store/Pipfile

```diff
@@ -0,0 +1,28 @@
+[[source]]
+
+url = "https://pypi.python.org/simple"
+verify_ssl = true
+name = "pypi"
+
+
+[packages]
+
+django = "~=2.0.5"
+django-debug-toolbar = "~=1.9.1"
+djangorestframework = "~=3.8.2"
+"psycopg2-binary" = "*"
```

Comment:
why this does package name have double quotes and the others haven't? :thinking:

Reply:
No idea was generated by pipenv... 🤔 ...will remove quotes.